core/chains/evm/log/integration_test.go

```diff
@@ -1859,15 +1829,11 @@ type simpleLogListener struct {
 func (helper *broadcasterHelper) newLogListenerWithJob(name string) *simpleLogListener {
 	t := helper.t
 	db := helper.db
-	jb := &job.Job{
-		Type:          job.Cron,
-		SchemaVersion: 1,
-		CronSpec:      &job.CronSpec{CronSchedule: "@every 1s"},
-		PipelineSpec:  &pipeline.Spec{},
-		ExternalJobID: uuid.New(),
-	}
-	err := helper.pipelineHelper.Jrm.CreateJob(testutils.Context(t), jb)
-	require.NoError(t, err)
+
+	var cronID int32
+	require.NoError(t, db.Get(&cronID, `INSERT INTO cron_specs (cron_schedule,created_at,updated_at) VALUES ('zzz', NOW(), NOW()) RETURNING id`))
+	var jobID int32
+	require.NoError(t, db.Get(&jobID, `INSERT INTO jobs (schema_version,type,external_job_id,created_at,cron_spec_id) VALUES (1, 0, $1, NOW(), $2) RETURNING id`, uuid.New(), cronID))
```

Comment:
Is there another way we can do this? Running database inserts directly in a unit test seems like it could be very brittle over the long term.

Reply:
This is only to support the transition, because core still owns the whole DB schema. Once EVM owns it's own DB schema, then we won't have to do this. (and this package might go away before then anyways)

Reply:
Ah ok. Thanks